Tests/SwiftBasicFormatTest/BasicFormatTests.swift

```diff
@@ -250,6 +250,23 @@ final class BasicFormatTest: XCTestCase {
     )
   }
 
+  func testMultilineStringLiteralWithBlankLines() {
+    assertFormatted(
+      source: #"""
+        assertionFailure("""
+          First line
```

Comment:
The blank line is gone. The code is the same as the test code above `testMultilineStringLiteral()`.

Reply:
Oh, copying code from GitHub apparently drops empty lines 🤯 Thanks for catching it. I updated the test case